support-lambdas/acquisitions-firehose-transformer/build.sbt

```diff
@@ -5,17 +5,20 @@ name := "acquisitions-firehose-transformer"
 description:= "A Firehose transformation lambda for serialising the acquisitions event stream to csv"
 
 libraryDependencies ++= Seq(
-  "com.amazonaws" % "aws-lambda-java-core" % "1.2.0",
   "com.amazonaws" % "aws-lambda-java-events" % "2.2.4",
+  "software.amazon.awssdk" % "auth" % "2.17.19",
+  "software.amazon.awssdk" % "dynamodb" % "2.17.19",
```

Comment:
Does Scanamo not pull these in canonically?

Reply:
good point, I can remove `auth` and `dynamodb`!